py3status/cli.py

```diff
@@ -33,109 +33,108 @@ def parse_cli():
+        help="enable debug logging in syslog and --log-file",
     )
     parser.add_argument(
         "-g",
         "--gevent",
         action="store_true",
         default=False,
         dest="gevent",
-        help="enable gevent monkey patching (default False)",
+        help="enable gevent monkey patching",
     )
     parser.add_argument(
         "-i",
         "--include",
+        metavar="PATH",
         action="append",
         dest="include_paths",
-        help=(
-            "include user-written modules from those "
-            "directories (default ~/.i3/py3status)"
-        ),
+        help="append additional module paths",
     )
     parser.add_argument(
         "-l",
         "--log-file",
+        metavar="FILE",
         action="store",
         dest="log_file",
         type=str,
+        help="enable logging to FILE",
         default=None,
-        help="path to py3status log file",
+    )
+    parser.add_argument(
+        "-m",
+        "--disable-click_events",
+        action="store_false",
+        dest="click_events",
+        help="disable click events",
     )
     parser.add_argument(
         "-n",
         "--interval",
+        metavar="INT",
         action="store",
         dest="interval",
         type=float,
+        help="refresh interval for py3status (default %(default)s)",
         default=1,
-        help="update interval in seconds (default 1 sec)",
     )
     parser.add_argument(
-        "-s",
-        "--standalone",
-        action="store_true",
-        help="standalone mode, do not use i3status",
+        "-s", "--standalone", action="store_true", help="run py3status without i3status"
     )
     parser.add_argument(
         "-t",
         "--timeout",
+        metavar="INT",
         action="store",
         dest="cache_timeout",
         type=int,
+        help="injection cache timeout (default %(default)s)",
         default=60,
-        help="default injection cache timeout in seconds (default 60 sec)",
     )
     parser.add_argument(
-        "-m",
-        "--disable-click-events",
+        "-v",
+        "--version",
         action="store_true",
-        dest="disable_click_events",
-        default=False,
-        help="disable all click events",
-    )
-    parser.add_argument(
-        "-v", "--version", action="store_true", help="show py3status version and exit"
+        help="show py3status version and exit",
     )
+
     # FIXME we should make all of these explicit so they self document etc
     parser.add_argument("cli_command", nargs="*", help=argparse.SUPPRESS)
-
     options = parser.parse_args()
 
-    # only asked for version
     if options.version:
-        import sys
         from platform import python_version
         from py3status.version import version
 
-        print("py3status version {} (python {})".format(version, python_version()))
-        sys.exit(0)
+        print("py3status {} (python {})".format(version, python_version()))
```

Comment:
This is the sort of thing that should be in the initial PR comment because this is a change of behaviour, user experience, breaks scripts relying on this etc.  We must be explicit about things like this.

Reply:
What do you want to do here? Keep old or new?

Reply:
I am keeping it new. I am wondering about `py3status -v` vs `py3-cmd -V` vs `py3-cmd -v`. What would make more sense... to use `py3status -V`, `py3-cmd -V`... for version... or verbose? I wanted both of them with same `v/V` letter.